nexus/src/db/datastore.rs

```diff
@@ -40,8 +40,10 @@ use crate::db::lookup::LookupPath;
 use crate::db::model::DatabaseString;
 use crate::db::model::IncompleteVpc;
 use crate::db::model::Vpc;
+use crate::db::queries::network_interface::DeleteNetworkInterfaceError;
+use crate::db::queries::network_interface::DeleteNetworkInterfaceQuery;
+use crate::db::queries::network_interface::InsertNetworkInterfaceError;
 use crate::db::queries::network_interface::InsertNetworkInterfaceQuery;
```

Comment:
Nit, since there are a stack of these, if you want to decrease verbosity, I find:

```
db::queries::network_interface::DeleteError
...network_interface::DeleteQuery
...network_interface::InsertError
...network_interface::InsertQuery
```

just as readable as the current version - the "NetworkInterface" bit is already implied by the namespace

Reply:
Yep, will do.